src/server/api/controllers/eventsNews.js

```diff
@@ -0,0 +1,184 @@
+import { badgerBrain } from '../../../shared/config';
+
+const badgerBrainEndpoint =
```

Comment:
I think it would be nicer if this was an env var rather than a flag so we can point it at any instance we want.